cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -19,6 +20,9 @@
 {% if pillar['enable_deployments'] is defined -%}
   {% set enable_deployments = "--enable-deployment-controller=" + pillar['enable_deployments'] -%}
 {% endif -%}
+{% if grains.cloud in [ 'gce' ] and grains.cloud_config is defined -%}
```

Comment:
@kubernetes/rh-cluster-infra @smarterclayton @kubernetes/goog-control-plane any objections to make this the flag's default?